src/main/java/duckling/responders/FileContents.java

```diff
@@ -0,0 +1,56 @@
+package duckling.responders;
+
+import duckling.requests.Request;
+import duckling.ResponseHeaders;
+
+import java.io.*;
+
+import static java.net.URLConnection.guessContentTypeFromName;
+import static java.net.URLConnection.guessContentTypeFromStream;
+
+public class FileContents extends Responder {
+    private File file;
+
+    public FileContents(Request request, OutputStream outputStream) {
+        super(request, outputStream);
+
+        this.file = request.getFile();
+    }
+
+    @Override
+    public boolean matches() {
+        return this.file.exists() && !this.file.isDirectory();
+    }
+
+    @Override
+    public void respond() throws IOException {
```

Comment:
I've been brewing on a way to get the `OutputStream` use decoupled from individual `Responder` objects, TBH.  This might be a good enough opportunity to do that.